### PR TITLE
open-cluster-management/backlog#15693 review

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/cluster_proxy_addon.adoc [hunk 1]
```diff
@@ -1,100 +1,104 @@
...
++
+----
+sed -i'' -e '/client-certificate-data/d' cluster-proxy.kubeconfig
+sed -i'' -e '/client-key-data/d' cluster-proxy.kubeconfig
+----
+
+... Add the token of the service account:
++
+----
+sed -i'' -e '/token/d' cluster-proxy.kubeconfig
```
Comment (xuezhaojun) [inline clusters/cluster_proxy_addon.adoc]: This one is to delete the old token, so it's more proper to be in the "Delete the original user credentials by entering the following commands"

Comment (cadawson) [reply clusters/cluster_proxy_addon.adoc]: Moved this

**Review by xuezhaojun — COMMENTED** [review]
/lgtm

**Review by swopebe — APPROVED** [review]

**Comment by swopebe** [comment]
Looks very good, I made a small change.

**Pull request merged**